process/charter.html

```diff
@@ -82,262 +82,77 @@ <h4>Tools and templates</h4>
 href="https://www.w3.org/Consortium/Process/#WGCharter">list
-of charter requirements</a> to ensure that your charter is complete
-(e.g., it includes a duration for the group).</p>
+	of charter requirements</a>.</p>
 
-<p class=timeline><em>Timeline:</em> For existing groups, this takes no time. For new groups, it may take one month or more for W3M
-  to start dedicating resources.</p>
+<p>&nbsp;</p>
 
-<h2 id="advance">2. Advance Notice</h2>
+<h2 id="advance">1. New Groups: Advance Notice</h2>
 
 <!-- Resolved at W3M 11 July 2007 -->
 
-<p>W3C Process (<a
-href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">section
-"Working Group and Interest Group Charter Development"</a>) requires that the Team notify the Advisory Committee when a
+<p><a
+href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">W3C Process</a> requires Team notify the Advisory Committee when a
 charter for a <strong>new</strong> Working Group or Interest Group is
 in development. The exact timing of the advance notice may vary from
 charter to charter. In practice, if the advance notice would precede
 the formal call for review by only a short delay, we skip the advance
-notice; see the <a href="https://www.w3.org/new-doc-from-template?location=%2FTeam%2F&amp;template=%2Fafs%2Fw3.org%2Fpub%2FWWW%2FTeam%2FTemplates%2Fadv-charter.html&amp;submit=Continue...">Advance Notice for Charter template</a></p>
+notice.</p>
 
-<p>The Team should consult <a href="/2005/09/adv-notice.html">How to
+<p>Consult <a href="/2005/09/adv-notice.html">How to
 Send Advance Notice of Work to the Advisory Committee</a> for more
-information.</p>
+information and a template announcement.</p>
 
-<p class=timeline><em>Timeline:</em> This takes a week at most.</p>
+	<h2 id="minor">2. Existing Groups: Minor Changes and Extensions</h2>
 
+	<p>The Chair(s) and Group may make requests for minor changes and extension via their Team Contacts(s), who will forward the requests to the Project and Strategy Leads.  Include a justification, which might be as simple as "we need three more months to complete our deliverable commitments and are fairly certain we can complete them within those three months."</p>
+	
 <h2>3. <a id="creation" name="creation">Charter Creation</a></h2>
 
-<p>The Team Contact works with the Strategy Lead to
-draft a provisional charter. See the
-<a href="https://w3c.github.io/charter-drafts/charter-template.html">Charter template</a>
-and
-W3C Process Document for the
-<a href="https://www.w3.org/Consortium/Process/#WGCharter">list of charter requirements</a>.
-The Team Contact may wish to share drafts of the provisional
-charter with a candidate Chair (if one exists), a relevant interest
-group or interested community of W3C members prior to Member
-review.</p>
-
-<p class=timeline><em>Timeline:</em> due to the discussion needing to happen with the community and the need
-  for horizontal review, this takes 1.5 to 4 months for existing groups.</p>
```

Comment:
This time estimates is useful for team contacts of far in advance to prepare their materials. It was added following a request from W3M so I'd like to keep in some form.

Reply:
added this back in